### ISSUE TITLE
JSON Matchers does not offer "Click to see difference"

### ISSUE THREAD

**Issue by busches** [kotest/kotest-intellij-plugin#155]
When using a JSON matcher and the assertion fails, there is no "Click to see difference" option in IntelliJ to quickly compare them.

Working example:
`401 shouldBe 402`
![image](https://user-images.githubusercontent.com/857700/140092553-1d6ecfea-9375-4522-bd2e-c8b43aaae5d8.png)

JSON Example:
`"""{"a":1}""" shouldMatchJson """{"a":2}"""`
![image](https://user-images.githubusercontent.com/857700/140092659-8cd4d4c4-c71a-480c-b32f-c96f980a75e5.png)

This is very useful when comparing larger JSON objects.

Comparing the output, it looks like the JSON example values need to be wrapped in `<X>`.

**Comment by Kantis** [comment]
Is this with latest 5.0.0 milestone build?

**Comment by busches** [comment]
No, using latest 4.6.3. From other conversations it doesn't sound like 5 will be out for sometime.

**Comment by Kantis** [comment]
Right 😄 Was just trying to figure out if we might've fixed this one already. I'll have a look. Should definitely be fixed if not!

**Update:** Okay, seems to not work. Will try to get a fix in with 5.0.
